lib/qops/environment.rb

```diff
@@ -26,17 +28,35 @@ def self.print_with_colour(message, level = :normal)
         puts Rainbow(message).bg(:black).red
       when :warning
         puts Rainbow(message).bg(:black).yellow
+      when :good
+        puts Rainbow(message).bg(:black).green
       else
         puts message
       end
     end
 
-    def initialize(profile: nil, force_config: false)
+    def initialize(profile: nil, force_config: false, verbose: false)
       @_aws_config = { region: configuration.region }
       @_aws_config[:profile] = profile unless profile.nil?
+
       @_force_config = force_config
-      profile.nil? ? opsworks.config.credentials.credentials : @_aws_config[:profile] = profile
-      puts Rainbow("using aws profile #{profile}").bg(:black).green unless profile.nil?
+      @_verbose = verbose
+
+      if profile.nil?
+        opsworks.config.credentials.credentials
```

Comment:
what is this line supposed to be doing?

Reply:
thats a very good question. To be honest I forget. It was there before I simply refactored.

Reply:
Oh yeah I saw that line of code in the diff before and after, so thats why I didn't put the status as `Request Changes`, but I thought it worth noting.